test/e2e/helpers/wallet/common-wallet-values.ts

```diff
@@ -4,6 +4,9 @@ import * as fs from 'fs'
 //URLS
 const chromeExtensionURL: string = 'chrome-extension://jfaancmgehieoohdnmcdfdlkblfcehph/index.html'
 export const testDAppUrl = 'https://vegaprotocol.github.io/vegawallet-browser/'
+export const consoleSmokePublicKey = '10743917237e3f76eabcec06acc4e56807c468c6a84f437c4c9ff75dc2822851'
```

Comment:
will move these before this goes down, just seen I have put them under //URLS, should probably make into an object instead of comments.